Neos.Neos/Resources/Private/Templates/Service/Nodes/Index.html

```diff
@@ -35,4 +35,5 @@ <h1>{neos:backend.translate(id: 'service.nodes.title', value: 'Nodes')}</h1>
         </div>
     </body>
 </html>
-<f:section name="breadcrumb"><f:if condition="{node.parent} && {node.parent.depth} > 1"><f:render section="breadcrumb" arguments="{node: node.parent}" /> &gt; </f:if>{node.label}</f:section>
+<f:section name="breadcrumb"><f:if condition="{node.parent} && {node.parent.depth} > 1"><f:render section="breadcrumb" arguments="{node: node.parent}" /> &gt; </f:if>{neos:node.label(node: node)}</f:section>
```

Comment:
haha there is no `node.parent` lol :D

Reply:
Nope, clearly not, oversight, I can do another fix for that.

Reply:
yeah dont worry, https://github.com/neos/neos-development-collection/issues/5023 is the full stuff